pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -515,23 +515,23 @@ type ETCDController struct {
 	// Workers specify number of worker threads in ETCD controller
 	// Defaults to 3
 	// +optional
-	Workers *int64 `json:"etcdControllerWorkers,omitempty"`
+	Workers *int64 `json:"workers,omitempty"`
 }
 
 // CustodianController contains config specific to custodian controller
 type CustodianController struct {
 	// Workers specify number of worker threads in custodian controller
 	// Defaults to 3
```

Comment:
Same as above:

https://github.com/gardener/gardener/blob/c52af1b21cc9a7340c1ca2574f107a2524a5c94c/pkg/operation/botanist/component/etcd/bootstrap.go#L334-L336

`3` in doc string vs `10` used in the component itself.